index.ts

```diff
@@ -301,17 +303,20 @@ collect.set('isRepoPRList', [
 	'https://github.com/sindresorhus/refined-github/pulls?q=is%3Apr+is%3Aclosed',
 ]);
 
-// `issues/fregante` is a list but `issues/1`, `issues/new`, `issues/new/choose`, `issues/templates/edit` aren’t
 export const isRepoIssueList = (url: URL | HTMLAnchorElement | Location = location): boolean =>
-	Boolean(getRepo(url)?.path.startsWith('issues')) &&
-	!/^issues\/(\d+|new|templates)($|\/)/.test(getRepo(url)?.path!);
+	// `issues/fregante` is a list but `issues/1`, `issues/new`, `issues/new/choose`, `issues/templates/edit` aren’t
+	/^issues(?!\/(\d+|new|templates)($|\/))/.test(getRepo(url)?.path!) ||
```

Comment:
Merged the two tests in a single one, but I don't know how you feel about look-aheads in regexes :smile:

Reply:
Will break safari

Reply:
> Will break safari

From https://caniuse.com/?search=lookahead:
> Lookahead assertions are part of JavaScript's original regular expression support and are thus supported in all browsers.

You're thinking about _lookbehind_ assertions: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#browser_compatibility

Reply:
You are correct, my mistakr

Reply:
Why lookaheads? You just need to prepend `^labels\/^|` to the regex

Reply:
> Why lookaheads?

To exclude a few specific URLs among those starting with `issues/`. The negative lookahead allows to merge the `startWith` condition into the original regex (that's what I meant by "merging the two tests", my bad if that was confusing). The label stuff is treated separately.

> You just need to prepend ^labels\/| to the regex

To which regex ? The original one ? I don't see how that would work since it's just here to exclude stuff :thinking:

Reply:
https://github.com/fregante/github-url-detection/pull/75/commits/eea00e7407c2c29f993d128604cb6db8a18f0f4d